### DISCUSSION TITLE
magick montage Sequence [1-10] returns the same result as [1,5,10] , or [5,10,1] etc?

### DISCUSSION THREAD

**Discussion by internet** [ImageMagick/ImageMagick#1958] [General] [answered]
I'm trying to create a montage that skips every nth frame of a sequence, and having no luck. Am I misunderstanding the image sequence syntax? 
example sequence would 
C:/filename/fake.1001.jpeg
C:/filename/fake.1002.jpeg
C:/filename/fake.1003.jpeg etc to say ten frames. 

magick montage "C:/filename/fake.%04d.jpeg[1-10]" -geometry 240x128x0 -tile x1 -format jpeg -quality 100 C:/filename/fake_montage.jpeg

this creates a montage of ten frames in a row. 

however  "C:/filename/fake.%04d.jpeg[1,5,10]" also provides 1 though 10, where my expectation would be a montage of frames 1001, 1005, 1010. 

Am I doing something wrong?

Windows 10 pro, ImageMagick-7.0.10-10-portable-Q16-x64

thanks!

www

**Comment by snibgo** [comment]
Your files are numbered 1001, 1002 etc, and those are the numbers you should use in the brackets [].

**Reply by internet** [reply]
Thanks for the quick reply! 

I just tried this, and had the same results. Basically any combination of numbers within the brackets will use all frames (in order) within the min and max values displayed
eg [1007,1001] will provide 1001,1002,1003,1004,1005,1006,1007

**Comment by fmw42** [comment, marked as answer]
Write a script loop with your OS to select only the frames you want and put them in a string list or text file. Then input the string or text file as the list of files you want in montage or pipe them from convert to montage via stdout and stdin.

**Comment by internet** [comment]
SUCCESS! 

Followup here, for other poor Windows users:

- create a list of files you want in a text file. JUST the basename, eg fake.1001.jpeg NOT C:/filename/fake.1001.jpeg
- save this text file in the same folder as the sequence. eg C:/filename/files.txt
- from the cmd shell, browse to that same folder. This is important. 
- then run it like this:  magick montage @**files.txt** -geometry 240x128x0 -tile x1 -format jpeg -quality 100 C:/filename/fake_montage.jpeg 
NOTE: You can't use the full path here (eg @C:/filename/files.txt) or you'll get bloberrors, as something in IM tries to concatenate the names and adds a surplus @C: drive letter to the names. 

I found the solution to this in the old forums;
https://www.imagemagick.org/discourse-server/viewtopic.php?t=34634 
I note the two people credited in solving this on the old forum share your usernames, so thank you both, twice!
